worker/src/main/java/gov/cms/ab2d/worker/config/AuditJobSetup.java

```diff
@@ -0,0 +1,57 @@
+package gov.cms.ab2d.worker.config;
+
+import gov.cms.ab2d.audit.job.FileDeletionJob;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.*;
+import org.quartz.impl.StdSchedulerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+import static org.quartz.CronScheduleBuilder.cronSchedule;
+import static org.quartz.JobBuilder.newJob;
+import static org.quartz.TriggerBuilder.newTrigger;
+
+@Slf4j
+@Component
+public class AuditJobSetup {
+
+    @Value("${audit-files-cron}")
+    private String auditFilesCron;
+
+    @Value("${efs.mount}")
+    private String efsMount;
+
+    @Value("${audit-files-ttl-hours}")
+    private int auditFilesTTLHours;
+
+    public static final String JOB_NAME = "DeleteAuditFilesJob";
+
+    public static final String CRON_GROUP = "FileDeletion";
+
+    public static final String TRIGGER_NAME = "DeleteAuditFilesTrigger";
+
+    @PostConstruct
+    public void setupAuditJob() throws SchedulerException {
+        SchedulerFactory sf = new StdSchedulerFactory();
```

Comment:
You can avoid dealing with low-level Quartz classes if you use Spring's facilities. Take a look at `QuartzSetup` that @spathiyil wrote.